tests/std/tests/P0408R7_efficient_access_to_stringbuf_buffer/test.cpp

```diff
@@ -0,0 +1,58 @@
+#include <cassert>
+#include <memory_resource>
+#include <sstream>
+#include <string>

```

Comment:
This should include `<utility>` for `move()`. It should also probably include `<string_view>` because it's being compared (even though not directly named).